### PR TITLE
Issue #5508 - Revived ANTIALIAS keyword in STYLE section 

### PR DESCRIPTION
**Pull request by erik-h11**
Revived ANTIALIAS keyword in STYLE section (only for line rendering). 

The parser in the current code still recognizes this keyword, but the loadStyle() method simply ignores it. I'm now setting a flag in styleObj if the symbol is MS_FALSE, then copy that flag to strokeStyleObj, and in agg2RenderLine (mapagg.cpp) I pick renderer_scanline_bin_solid if the flag is MS_FALSE.

### CODE REVIEW

File: mapagg.cpp [hunk 1]
```diff
@@ -200,7 +202,8 @@ int agg2RenderLine(imageObj *img, shapeObj *p, strokeStyleObj *style)
 
   AGG2Renderer *r = AGG_RENDERER(img);
   line_adaptor lines = line_adaptor(p);
-
+  
+ 
 #ifdef AGG_ALIASED_ENABLED
```
Comment (tbonfort) [inline mapagg.cpp]: please remove this unused ifdef'd block

File: mapagg.cpp [hunk 2]
```diff
@@ -146,6 +147,7 @@ class AGG2Renderer
   renderer_base m_renderer_base;
   compop_renderer_base m_compop_renderer_base;
   renderer_scanline m_renderer_scanline;
+  renderer_scanline_aliased m_renderer_scanline_aliased;
 #ifdef AGG_ALIASED_ENABLED
```
Comment (tbonfort) [inline mapagg.cpp]: please remove this unused block

**Comment by erik-h11** [comment]
Yes, this change affects lines with width > 1.
I added two test files; not sure how to insert them into msautotest framework.

**Comment by rouault** [comment]
Superesed per https://github.com/MapServer/MapServer/pull/6225

**Pull request closed without merging**